App/stores/location.tsx

```diff
@@ -109,6 +144,13 @@ export function LocationContextProvider({
 		)().catch(sentryError('LocationContextProvider'));
 	}, []); // eslint-disable-line react-hooks/exhaustive-deps
 
+	useEffect(() => {
+		AsyncStorage.setItem(
+			'LAST_KNOWN_LOCATION',
```

Comment:
This hardcoded string appears string. Could you refactor it out in a `const LAST_KNOWN_LOCATION_KEY = 'LAST_KNOWN_LOCATION';` ?

Reply:
yes indeed